### ISSUE TITLE
Auto HDR Gaming

### ISSUE THREAD

**Issue by juturna01** [dylanraga/win11hdr-srgb-to-gamma2.2-icm#63]
I saw a previous post in here from months ago discussing the SDR content brightness slider/these profiles/how it translates to in game paper white etc, but it didn't seem to reach a conclusion? Or I just didn't understand something. Point being, I'm confused as heck and was wondering if you figured something out there yet XD I have no idea what profile I should be using that you have posted or what to set my SDR slider to for auto HDR gaming. I have a LG C2 btw!

**Comment by Animesh-Does-Code** [comment]
Are you talking about Issue #19? From what I understand according to [this](https://github.com/dylanraga/win11hdr-srgb-to-gamma2.2-icm/issues/19#issuecomment-1855906593) comment from Filoppi, when your SDR content brightness slider is set to 0 (80 nits), Auto HDR will require a gamma correction profile made for an SDR slider value of 43 (~250 nits). I assume Auto HDR intensity also has to be at 100. If someone understands this better, feel free to correct me if I got this wrong.

Unfortunately, there's no color profile for exactly 250 nits SDR brightness on this GitHub page, but I put together a quick command line tool to set up gamma correction using AutoHotkey and dispwin: https://github.com/Animesh-Does-Code/win11hdr-srgb-to-gamma2.2-ahk (Credit goes to Dylan Raga and mspeedo for the vast majority of the work).

So, to correct gamma for Auto HDR accurately, first you have to set your SDR content brightness slider to '0'. Then, in the setup process of the above tool, enter `43` when it asks for your SDR content brightness slider value, and `2.2` for the gamma. 

Once the script is running, try an Auto HDR game and see how it looks. You can toggle the gamma correction off and on with the shortcuts `Win+F1` and `Win+F2` respectively. Don't use any of the color profiles alongside this, though.

**Comment by juturna01** [comment]
Yup that's the one! This is wild haha. I'll certainly try that and see how it looks at least, but I imagine that will be quite bright. I can't wait for the day none of this is an issue 🙏
